batchflow/models/torch/base.py

```diff
@@ -1709,14 +1708,11 @@ def save(self, path=None, use_onnx=False, path_onnx=None, use_openvino=False, pa
         kwargs : dict
             Other keyword arguments, passed directly to :func:`torch.save`.
         """
-
-        if path is None:
-            dirname = os.getcwd()
-            path = os.path.join(dirname, "model.pt")
-        else:
-            dirname = os.path.dirname(path)
-            if dirname and not os.path.exists(dirname):
-                os.makedirs(dirname)
+        available_formats = ("pt", "onnx", "openvino", "safetensors")
+        assert format in available_formats, f"Format must be in {available_formats}"
```

Comment:
You can use `assert` to check the arguments in internal methods, but in public methods you need to raise `ValueError`